RELEASE.md

```diff
@@ -1,3 +1,418 @@
+- `strike` attribute of type `OptionStrike` added to `OptionPayout` type.
+- `exerciseTerms` attribute of type `ExerciseTerms` added to `OptionPayout` type.
+- `exerciseTerms` attribute of type `OptionExercise` removed from `OptionPayout` type.
+- `americanExercise`, `europeanExercise`, and `bermudaExercise` attributes of types `AmericanExercise`, `EuropeanExercise`, and `BermudaExercise` respectively removed from `CancelableProvision`, `ExtendibleProvision`, and `OptionalEarlyTermination` types.
+- `exerciseTerms` attribute of type `ExerciseTerms` added to `CancelableProvision`, `ExtendibleProvision`, and `OptionalEarlyTermination` types.
+-  Added new `ExerciseTerms` type with all of the distinct fields present before in `AmericanExercise`, `EuropeanExercise`, and `BermudaExercise` types.
+
+_Enumerations_
+
+- Added new `OptionExerciseStyleEnum` enumeration.
+
+- _Backward Incompatible Changes_
+
+
+- _Motivation for Change_
+
+This change modifies the `OptionPayout` structure to simplify it and reduce the redundancy and over-complexity of the model.
+
+Specifically:
+- It removes the `OptionStyle` type from the `exerciseTerms` field, along with the three option exercise types contained inside it (`AmericanExercise`, `EuropeanExercise`, `BermudaExercise`), and replaces it with the distinct group of fields required to represent any type of option style.
+- The type `OptionExercise` is removed from the model, and instead the type `ExerciseTerms` is used for the field `exerciseTerms`.
+- The `style` enumeration is incorporated into the `ExerciseTerms` type to differentiate between American, European, and Bermuda styles.
+- Finally, the `strike`, previously under `OptionStyle`, has been moved outside of `ExerciseTerms` and is located directly under `OptionPayout`.
+
+
+_Sample Impact_
+
+There are many samples impacted by this change, namely all the samples utilizing the `OptionPayout` structure. The impact on the three following samples (one European, one American, one Bermuda) is shown to visualize how the refactoring of the `OptionPayout` affects the structure of the CDM trades:
+
+- `eqd ex04 european call index long form`: the `OptionStyle` has been removed in favor of the the `style` = "European", and the relevant fields previously under `europeanExercise` (`expirationDate` and `expirationTimeType`). Additionally, the `strike` is moved from inside `exerciseTerms` to outside.
+
+From:
+```
+"optionPayout": [
+    {
+        ...
+        "exerciseTerms": {
+            "optionStyle": {
+                "europeanExercise": {
+                    "expirationDate": [
+                        {
+                            "adjustableDate": {
+                                "unadjustedDate": "2004-12-19",
+                                "dateAdjustments": {
+                                    "businessDayConvention": "NONE",
+                                    "meta": {"globalKey": "24a738"}
```

Comment:
Remove all meta / key from the JSON samples, they pollute the view and aren't necessary